index.html

```diff
@@ -0,0 +1,253 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <script src="scrtipt.js"></script>
+    <title>Portfolio</title>
+</head>
+
+<body>
+
+    <header class="beginning">
+        <nav class="head">
+            <div id="slidemenu" class="popup">
+                <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+                <a href="#">Home</a>
+                <a href="#">About</a>
+                <a href="#">portfolio</a>
+                <a href="#">Contact</a>
+            </div>
+            <div class="bruw blue">
+                <h2>My Logo</h2>
+            </div>
+            <div class="menu"><span style="font-size:30px;cursor:pointer" onclick="openNav()"><img src="images/Hamburger button.svg">
+            </div>
+            <div class="navigation_option">
+                <a class="#">Portfolio</a>
+                <a class="#">About</a>
+                <a class="#">Contact</a>

```

Comment:
Kindly consider this gif:

![horizontal scroll](https://user-images.githubusercontent.com/27709832/203333526-75a1e47a-6ab7-424e-909f-841309a2d8a7.gif)

- The website's mobile screen scrolls horizontally.
- The hamburger is required to be replaced with the **x** when the mobile menu is open and when clicked should close the mobile menu.